### PR TITLE
OW-744 unity renderer

### PR DESCRIPTION
**Pull request by gmininberg**

### CODE REVIEW

File: Unity/ios/VonageUnityIntegration/transformers.cpp [hunk 1]
```diff
@@ -64,14 +66,7 @@ namespace vonage {
...
             return;
         }
-        std::unique_ptr<uint8_t[]> augmented_data;
-        uint32_t augmented_size = 0;
-        if(decompressor_ && target_frame->video_frame_buffer()->IsAugmented()){
-            webrtc::AugmentedVideoFrameBuffer* augemnted_video_frame = static_cast<webrtc::AugmentedVideoFrameBuffer*>(target_frame->video_frame_buffer().get());
-            if(!decompressor_->decompress(augemnted_video_frame->GetAugmentingData(), augemnted_video_frame->GetAugmentingDataSize(), augmented_data, augmented_size)){
-                
-            }
-        }
```
Comment (joliveraortega) [inline Unity/ios/VonageUnityIntegration/transformers.cpp:74]: Are we not de compressing now?

File: Unity/ios/VonageUnityIntegration/transformers.cpp [hunk 2]
```diff
@@ -102,11 +97,19 @@ namespace vonage {
                            input_width,
                            input_heigth);
         
-        // Send converted ARGB video frame buffer to Unity
-        [FrameworkLibAPI setInputBufferCpp:in_argb_data.get() rgbSize:(input_num_bytes) augmentedBuffer:augmented_data.get() augmentedSize:augmented_size rotation:GetRotation(target_frame->rotation())];
+        int64_t start = rtc::TimeNanos();
+        webrtc::AugmentedVideoFrameBuffer* augemnted_video_frame = static_cast<webrtc::AugmentedVideoFrameBuffer*>(target_frame->video_frame_buffer().get());
```
Comment (joliveraortega) [inline Unity/ios/VonageUnityIntegration/transformers.cpp:101]:
```suggestion
        webrtc::AugmentedVideoFrameBuffer* augmented_video_frame = static_cast<webrtc::AugmentedVideoFrameBuffer*>(target_frame->video_frame_buffer().get());
```

File: Unity/ios/VonageUnityIntegration/transformers.cpp (hunk 1 above)
Comment (gmininberg) [reply Unity/ios/VonageUnityIntegration/transformers.cpp:74]: as we talked in the daily not needed. but if it does we will add it back

File: Unity/ios/VonageUnityIntegration/transformers.cpp (hunk 2 above)
Comment (joliveraortega) [reply Unity/ios/VonageUnityIntegration/transformers.cpp]: Thanks!

**Review by joliveraortega — APPROVED** [review]

**Pull request merged**